### ISSUE TITLE
Trouble getting the discord-irc to start

### ISSUE THREAD

**Issue by n0rt0x** [reactiflux/discord-irc#473]
Okay, I made a new Issue since the other one was closed anyway. Initially, I had 3 issues/head scratching things. The first one was running the `sudo npm install -g discord-irc` command in order to install the discord-irc. My output said that some peer dependencies need manually installation, but according to the discord.js GitHub page those dependencies are purely optional.

Anyhoo, giving the command as I mentioned above, I also get the following warning: `npm WARN deprecated snekfetch@3.6.4: use node-fetch instead /usr/bin/discord-irc -> /usr/lib/node_modules/discord-irc/dist/index.js`. So my question is if it's possible that npm will primary listen to node-fetch instead of snekfetch? I Googled a bit but couldn't find a proper way to do it.

Lastly, I get the following results when trying to actually launch the discord-irc with: 

`sudo discord-irc --config /config.json` : `
usr/lib/node_modules/discord-irc/node_modules/check-env/index.js:12
throw new Error('Missing environment variable ' + missing[0]);
^

Error: Missing environment variable CONFIG_FILE
at module.exports (/usr/lib/node_modules/discord-irc/node_modules/check-env/index.js:12:13)
at Object.run [as default] (/usr/lib/node_modules/discord-irc/dist/cli.js:59:59)
at Object. (/usr/lib/node_modules/discord-irc/dist/index.js:12:27)
at Module._compile (internal/modules/cjs/loader.js:738:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
at Module.load (internal/modules/cjs/loader.js:630:32)
at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
at Function.Module._load (internal/modules/cjs/loader.js:562:3)
at Function.Module.runMain (internal/modules/cjs/loader.js:801:12)
at internal/main/run_main_module.js:21:11`

I have honestly no clue where the problem lies as my programming experiences are almost zero. Maybe the problem is something hilariously simple, but frankly, i don't see it atm. Note that I use Raspbian with latest updates and the latest version of Node.js v11.10.1 for this IRC bot.

So yeah, I was wondering if anyone could help me out on this one, would be very much appreciated!

**Comment by n0rt0x** [comment]
If my full config file is required for the troubleshooting, please let me know.

**Comment by n0rt0x** [comment]
@Throne3d perhaps you have any idea what's going on here?

**Comment by n0rt0x** [comment]
Could it be because of this? https://npm.community/t/npm-pack-leaving-out-files-6-8-0-only/5382

**Comment by n0rt0x** [comment]
I was also wondering if it's possible to let the bot to messsage HostServ to request a vhost upon connecting and automatically turn it on once the vhost is approved..

**Comment by ekmartin** [comment]
This error happens if you don't pass in either a `--config config.json` argument or set the config with an environment variable, e.g., `CONFIG_FILE=config.json discord-irc`. It looks like you're using the former though, which is strange. Maybe try with the environment variable and see what happens? 

The warning about `snekfetch` is completely harmless.

**Comment by n0rt0x** [comment]
Hi Martin, thanks for replying! I do use the newest configuration template. Anyhoo, i got the bot actually to work last night but i was too tired to update it here haha. I still do have to register the bot tho, so some minor changes in the config file (does that also explains that the bot only shows itself in Discord?)

**Comment by n0rt0x** [comment]
Okay scrap that. The bot is now fully working! 😃 Should this still work at this date for the vhost? https://github.com/reactiflux/discord-irc/issues/233

**Comment by Throne3d** [comment]
Should do! I assume you got everything working, so I'll close this – sorry, I must have missed it.
